boskos/cmd/cli/cli_test.go

```diff
@@ -416,9 +465,32 @@ failed to send heartbeat for resource "87527b0c-eac2-4f83-9a03-791b2239e093": st
 				t.Errorf("%s: request %d: incorrect content-type header, expected %s, saw %s", testCase.name, i, expected, actual)
 			}
 
+			if expected, actual := testCase.expectedCalls[i].header.Get("Authorization"), request.header.Get("Authorization"); expected != actual {
+				t.Errorf("%s: request %d: incorrect Authorization header, expected %s, saw %s", testCase.name, i, expected, actual)
+			}
+
 			if expected, actual := testCase.expectedCalls[i].body, request.body; !reflect.DeepEqual(expected, actual) {
 				t.Errorf("%s: request %d: incorrect body: %s", testCase.name, i, diff.StringDiff(string(expected), string(actual)))
 			}
 		}
 	}
 }
+
+func getMsg(s string) string {
```

Comment:
NACK to these changes

Reply:
Since logrus is removed, no need for those silly functions.